src/components/navigation/mobile-menu.tsx

```diff
@@ -25,17 +30,19 @@ export function MobileMenu(props: MobileMenuProps) {
   )
 }
 
-// TODO: needs type declaration
-type EntryProps = any
+interface MobileMenuLink {
+  childKey?: string
+  isChild?: boolean
+}
 
 function Entry({
   url,
   title,
+  icon = undefined,
```

Comment:
That shouldn't be needed since this is the default JS behavior (keys that aren't set are `undefined`)